### PR TITLE
User Story 60501: 101-stream-analytics-job

### PR DESCRIPTION
**Pull request by TomArcherMsft**
Part of POC to test generating sample code and articles using OpenAI.

### CODE REVIEW

File: quickstart/101-traffic-manager-external-endpoint/main.tf [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  resource_group_name                      = azurerm_resource_group.rg.name
+  location                                 = azurerm_resource_group.rg.location
+  streaming_units                          = var.number_of_streaming_units
+  events_out_of_order_max_delay_in_seconds = 0
+  events_late_arrival_max_delay_in_seconds = 5
+  data_locale                              = "en-US"
+  events_out_of_order_policy               = "Adjust"
+  output_error_policy                      = "Stop"
+
+  transformation_query = <<QUERY
```
Comment (grayzu) [inline quickstart/101-traffic-manager-external-endpoint/main.tf:28]: Will this query actually work or do variables need to be added for Aliases?

**Review by grayzu — APPROVED** [review]
Thanks for the PR!

One question but assuming the query is good as is, this should be good to go!

File: quickstart/101-traffic-manager-external-endpoint/main.tf (hunk 1 above)
Comment (TomArcherMsft) [reply quickstart/101-traffic-manager-external-endpoint/main.tf:28]: I ran the sample without a problem. I looked at the Bicep article (from which this article is based), but there's no info about those values.

https://learn.microsoft.com/en-us/azure/stream-analytics/quick-create-bicep?tabs=CLI

**Comment by TomArcherMsft** [comment]
@grayzu @lonegunmanb 

As @lonegunmanb mentioned, somehow I confused this PR with another. To avoid confusion, I'm closing this PR and will open a new one.

**Pull request closed without merging**